#!/usr/bin/env python
PACKAGE = "auv_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("linear_kp", double_t, 0, "Linear Kp Gain", 0.55, 0, 10)
gen.add("linear_kd", double_t, 0, "Linear Kd Gain", 0.2, 0, 10)
gen.add("angular_kp", double_t, 0, "Angular Kp Gain", 0.45, 0, 10)
gen.add("angular_kd", double_t, 0, "Angular Kd Gain", 0.2, 0, 10)

exit(gen.generate(PACKAGE, "auv_control", "AlignFrame"))
